crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -69,7 +69,12 @@ def _prepare_project_values(self, project, **kwargs):
         # Chronological list of sponsorships and fund donations for impact display
         impact = sorted(sponsorships + donations, key=lambda x: x["date"])
 
-        return {"project": project, "impact": impact}
+        fund = request.env['product.product'].sudo().search([
+            ('activate_for_crowdfunding', '=', True),
+            ("name", "like", project.product_id.product_tmpl_id.name)
+        ])
```

Comment:
I made the same suggestion in other PR.
```suggestion
        fund = project.product_id
```
You could directly access it from the template like that without needing to pass it as a separate variable.

Reply:
I will change the template later